### PR TITLE
fix: no articles redirect; no articles hide rail

### PR DESCRIPTION
**Pull request by ivan-badyulya**
JIRA
- [hide the articles rail on the overview tab if there are no articles. Redirect form the articles tab to overview if no articles.](https://artsyproduct.atlassian.net/browse/PX-4000)

### CODE REVIEW

File: src/v2/Apps/Partner/Routes/Overview/index.tsx [hunk 1]
```diff
@@ -15,15 +15,19 @@ const Overview: React.FC<OverviewProps> = ({ partner }) => {
     displayArtistsSection,
   } = partner
 
+  const AreArticlesExisted = !!articles.length
```
Comment (dzucconi) [inline src/v2/Apps/Partner/Routes/Overview/index.tsx:18]: I'd re-name/case this variable `const hasArticles = articles.length > 0` (or just inline it). The capitalized casing should be reserved for classes/components.

File: src/v2/Apps/Partner/partnerRoutes.tsx [hunk 2]
```diff
@@ -71,10 +71,37 @@ export const partnerRoutes: RouteConfig[] = [
...
+
+          if (!totalCount) {
+            throw new RedirectException(
+              `/partner2/${match.params.partnerId}`,
+              302
+            )
+          }
+
+          return <Component {...props} />
+        },
```
Comment (dzucconi) [inline src/v2/Apps/Partner/partnerRoutes.tsx:104]: 👍

**Review by dzucconi — APPROVED** [review]
Would just address that variable name. Feel free to merge when you're ready.

File: src/v2/Apps/Partner/Routes/Overview/index.tsx (hunk 1 above)
Comment (ivan-badyulya) [reply src/v2/Apps/Partner/Routes/Overview/index.tsx:18]: whoops, didn't notice it is PascalCased

**Pull request merged**